source/community/index.html.md

```diff
@@ -74,7 +74,7 @@ People using oVirt and all those who develop the software are welcome in [the \#
 
 If you don't already have a IRC client, you can install [one of the clients referenced here](http://www.irchelp.org/irchelp/new2irc.html), or join our channel in your web browser by using [mibbit](https://www.mibbit.com/).
 
-We're most active from about 07:00 UMT to 11:00 UMT (09:00 IST to 17:00 PST), but there are usually folks hanging out throughout the entire day. Also: be sure to [read our simple etiquette guidelines](Community_guidelines). (Quick summary: be nice!)
+We're most active from about 07:00 UMT to 11:00 UMT (09:00 IST to 17:00 PST), but there are usually folks hanging out throughout the entire day. Also: be sure to [read our simple etiquette guidelines](community-guidelines). (Quick summary: be nice!)
```

Comment:
I am not sure this is the right link above.
Here IS the right link: https://www.ovirt.org/community/about/community-guidelines/